lib/core/src/irods_error.cpp

```diff
@@ -136,6 +136,11 @@ const char *error::status_token_     = " status [";
         std::string  _fcn,
         const error& _rhs ) :
         error(_rhs) {
+ 
+        if (_msg != "") {
```

Comment:
Is this statement checking for emptiness? If so, the **empty** member function should be used. 
```
if (!_msg.empty()) {
    ...
}
```